src/main/scala/it/pagopa/interop/notifier/service/impl/QueueHandler.scala

```diff
@@ -47,8 +47,9 @@ class QueueHandler(
     _ = logger.debug("Organization id retrieved for message {} -> {}", msg.messageUUID, organizationId)
     nextEvent <- idRetriever.getNextEventIdForOrganization(organizationId)(m2mContexts)
     _ = logger.debug("Next event id for organization {} -> {}", nextEvent.organizationId, nextEvent.eventId)
-    _ = dynamoService.put(organizationId, nextEvent.eventId, msg)
-  } yield ()
+    result <- dynamoService.put(organizationId, nextEvent.eventId, msg)
+    _ = logger.debug("Message {} was successfully written to dynamodb", msg.messageUUID.toString)
```

Comment:
I know this is to keep the same approach, but if possible let's avoid using this positional implementation 🙏